integration/test_rdb_load_error_cleanup.py

```
@@ -0,0 +1,203 @@
+        # The bug window requires both: workers holding strong refs (parked)
+        # AND the main thread having unwound from LoadFromRDB (error logged).
+        # Either alone can pass without exercising the bug.
+        def error_logged():
+            return replica.does_logfile_contains(self.SIM_ERROR_LOG)
+
+        def workers_paused():
+            try:
+                n = replica.client.execute_command(
+                    "FT._DEBUG", "PAUSEPOINT", "TEST", self.PAUSEPOINT_NAME)
+                return int(n) >= 1
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)